app.js

```diff
@@ -38,7 +38,13 @@ var translator = new LanguageTranslatorV2({
 
 // render index page
 app.get('/', function(req, res) {
+  // If show_header is found in the query string and is set to 0 or false,
+  // the header will be hidden. 
+  var showHeader = (req.query.show_header && (
```

Comment:
Now that I look at the code, it will be better to call it `hide_header` so that the user specify `true` rather than using `show_header` which is `true` by default.

You can do something like
```js
res.render('index', {
    showHeader: !(req.query.showHeader == 'false' || req.query.showHeader == '0'),
    bluemixAnalytics: !!process.env.BLUEMIX_ANALYTICS,
  });
```